packages/workspace/src/browser/workspace-service.ts

```diff
@@ -115,16 +123,33 @@ export class WorkspaceService implements FrontendApplicationContribution {
     }
 
     protected async updateWorkspace(): Promise<void> {
+        if (this._workspace) {
```

Comment:
As per Theia coding guide lines, we should not use "_" in front of the private variable
 i.e.  "_workspace"
ref: https://github.com/theia-ide/theia/wiki/Coding-Guidelines#names

Reply:
Collision with https://github.com/theia-ide/theia/blob/ce1e855e8702dc009383c8630bafc45354e68df2/packages/workspace/src/browser/workspace-service.ts#L97

Reply:
It is common to have `_something` when you have a `something` getter in the same class.